tests/everest-core_tests/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+project(everest-core_tests
+    DESCRIPTION "Extending EVerest with testing modules/interfaces/types/errors.. etc"
+    LANGUAGES CXX C)
+
+# LTODO: why?
+# ev_add_project(${CMAKE_CURRENT_SOURCE_DIR} everest-core_tests)
```

Comment:
Why do we have the cmake option `CALLED_FROM_WITHIN_PROJECT`?
Shouldn't interfaces, types and errors installed in any case?

Reply:
@hikinggrass